src/components/content/catalog/services/details/serviceTemplateDetails.tsx

```diff
@@ -0,0 +1,68 @@
+import { CheckCircleOutlined, EnvironmentOutlined, SyncOutlined } from '@ant-design/icons';
+import { Card, Tag } from 'antd';
+import React from 'react';
+import catalogStyles from '../../../../../styles/catalog.module.css';
+import { ServiceTemplateDetailVo } from '../../../../../xpanse-api/generated';
+import { ServicePolicies } from '../policies/ServicePolicies';
+import ServiceDetail from './ServiceDetail';
+import { ServiceHostingOptions } from './ServiceHostingOptions';
+
+function serviceTemplateDetails({
+    isViewDisabled,
+    serviceDetails,
+    groupServiceTemplatesByCsp,
+    serviceCspInQuery,
+    serviceHostingTypeInQuery,
+    onChangeServiceHostingType,
+}: {
+    isViewDisabled: boolean;
+    serviceDetails: ServiceTemplateDetailVo;
+    groupServiceTemplatesByCsp: Map<string, ServiceTemplateDetailVo[]>;
+    serviceCspInQuery: string;
+    serviceHostingTypeInQuery: string;
+    onChangeServiceHostingType: (serviceTemplateDetailVo: ServiceTemplateDetailVo) => void;
+}): React.JSX.Element {
+    return (
+        <>
+            <Card className={catalogStyles.serviceTemplateCard}>
+                <div>
+                    {serviceDetails.isAvailableInCatalog ? (
+                        <Tag icon={<CheckCircleOutlined />} color='#87d068'>
+                            Available In Catalog
+                        </Tag>
+                    ) : (
+                        <Tag icon={<SyncOutlined spin />} color='#e67300'>
+                            Not Available in Catalog
+                        </Tag>
+                    )}
+                    {serviceDetails.isReviewInProgress ? (
+                        <Tag icon={<SyncOutlined spin />} color='#e67300'>
+                            Review In-Progress
+                        </Tag>
+                    ) : (
```

Comment:
we do not need a tag for reviewed. This will complicate things. 

Because a review can result in approve or reject. So a green tag is confusing. Also review can be for update or registration. So better we avoid this.